### DISCUSSION TITLE
Can't create a new project

### DISCUSSION THREAD

**Discussion by cesar-cardinale** [symfony/symfony#48228] [Q&A]
When executing this command :
`symfony new new_project --webapp`

I get this error :
```
Problem 1
    - symfony/flex[v2.0.0, ..., v2.2.3] require composer-plugin-api ^2.1 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
    - Root composer.json requires symfony/flex ^2 -> satisfiable by symfony/flex[v2.0.0, ..., v2.2.3].
```

What can I do, or need to upgrade to create a new project ?

Thanks !

**Comment by MatTheCat** [comment]
Guess this is duplicated by https://github.com/symfony/symfony/discussions/48229?

**Reply by cesar-cardinale** [reply]
Yes and it's solved, but don't know how to delete a discussion!

**Reply by MatTheCat** [reply]
I don’t think you can; just wanted to make sure!
